src/tensors/tensor_operators.h

```diff
@@ -200,13 +206,26 @@ static inline void LayerNormalizationGrad(
                             Tensor gamma,
                             Tensor beta,
                             float eps) {
-#ifdef CUDA_FOUND
   if(gradX->getBackend()->getDeviceId().type == DeviceType::gpu)
     gpu::LayerNormalizationGrad(allocator, gradX, gradGamma, gradBeta, adj, y, x, gamma, beta, eps);
   else
-#endif
     cpu::LayerNormalizationGrad(gradX, gradGamma, gradBeta, adj, y, x, gamma, beta, eps);
 }
+#else
+static inline void LayerNormalizationGrad(
```

Comment:
This feels like duplicating too much. What is the problem with the oruiginal? Can you solve it by saying `var;` to reference the unreferenced variables?

Reply:
See my comment above. Resolve conversation when satisfied.

Reply:
I like the MAYBE_UNUSED approach, as it seems low-overhead.